_posts/2020-07-29-galaxy-20-05.md

```diff
@@ -0,0 +1,43 @@
+---
+site: freiburg
+tags: [release]
+title: UseGalaxy.eu update to 20.05
+supporters:
+ - denbi
+ - elixir
+---
+
+We've updated our Galaxy server to **version 20.05**. This version includes the **migration to Python 3** and improves usability in the aspects listed below.
+
+
+For a full list of the new features
+please refer to the release notes of versions [20.01](https://docs.galaxyproject.org/en/master/releases/20.01_announce_user.html) and [20.05](https://docs.galaxyproject.org/en/master/releases/20.05_announce_user.html).
+
+
+### Tool Recommendations
+
+When you add a tool to your workflow in [UseGalaxy.eu](https://usegalaxy.eu/), you can now get a recommendation on which tools you can run next. This [feature](https://galaxyproject.eu/posts/2019/06/27/tool-prediction/) applies machine learning to data based on previous usage and was already deployed on the European Galaxy server since a year - now this feature was integrated into te upstream code and works better than ever.
```

Comment:
```suggestion
When you add a tool to your workflow in [UseGalaxy.eu](https://usegalaxy.eu/), you can now get a recommendation on which tools you can run next. This [feature](https://galaxyproject.eu/posts/2019/06/27/tool-prediction/) applies machine learning to data based on the previous usage and was already deployed on the European Galaxy server since a year - now this feature was integrated into the upstream code and works better than ever.
```